composer.json

```diff
@@ -17,7 +17,7 @@
     "behat/mink-selenium2-driver": "~1.1",
     "behat/behat": "~3.0,>=3.0.5",
     "behat/mink-extension": "~2.0",
-    "drupal/drupal-driver": "~1.0@dev"
+    "drupal/drupal-driver": "dev-language"
```

Comment:
This can be switched back now as I've merged https://github.com/jhedstrom/DrupalDriver/pull/45